### PR TITLE
ports "Better Supermatter Sounds (#45462)" from tg

### PR DESCRIPTION
**Pull request by ghost**
About The Pull Request

Replaces supermatter sounds with a new sound system for it.
Works based on having a constant drone that randomly plays accent noises over it.
https://www.youtube.com/watch?v=k5kKc5mpl-A&feature=youtu.be

all credits go to @actioninja i am only porting this to bee.
link to original PR tgstation/tgstation#45462

Why It's Good For The Game

the old sounds are terrible, this makes it actually sound and feel threatening as the radioactive doomsday crystal it is.
Changelog
🆑
soundadd: New supermatter sound system
/🆑

### CODE REVIEW

**Comment by BeloneX** [comment]
link to original PR https://github.com/tgstation/tgstation/pull/45462

**Comment by Lagomorphica** [comment]
this is really fucking hot and i want it

File: code/game/sound.dm [hunk 1]
```diff
@@ -165,4 +165,10 @@
 				soundin = pick('sound/items/bikehorn.ogg', 'sound/items/AirHorn2.ogg', 'sound/misc/sadtrombone.ogg', 'sound/items/AirHorn.ogg', 'sound/effects/reee.ogg',  'sound/items/WEEOO1.ogg', 'sound/voice/beepsky/iamthelaw.ogg', 'sound/voice/beepsky/creep.ogg','sound/magic/Fireball.ogg' ,'sound/effects/pray.ogg', 'sound/voice/hiss1.ogg','sound/machines/buzz-sigh.ogg', 'sound/machines/ping.ogg', 'sound/weapons/flashbang.ogg', 'sound/weapons/bladeslice.ogg')
 			if("goose")
 				soundin = pick('sound/creatures/goose1.ogg', 'sound/creatures/goose2.ogg', 'sound/creatures/goose3.ogg', 'sound/creatures/goose4.ogg')
+			if("warpspeed")
+				soundin = 'sound/effects/hyperspace_begin.ogg'
```
Comment (ike709) [inline code/game/sound.dm]: I don't see where you use `"warpspeed"` or where you add the ogg file.

Comment (ghost) [reply code/game/sound.dm]: > I don't see where you use `"warpspeed"` or where you add the ogg file.

me neither.

Comment (ike709) [reply code/game/sound.dm]: That... that means you should remove these two lines...

Comment (ghost) [reply code/game/sound.dm]: > 
> 
> That... that means you should remove these two lines...

done

**Comment by ike709** [comment]
Fix your CL

**Comment by ghost** [comment]
> 
> 
> Fix your CL

done

**Comment by ike709** [comment]
nope, still not correct

**Comment by ghost** [comment]
> nope, still not correct

how

**Comment by ike709** [comment]
Should be
/🆑
not
🆑

For the second one.

**Comment by qwertyquerty** [comment]
smh just edit it yourself ike

**Pull request merged**